### PR TITLE
Changes to fSCA and SNOTEL

### PR DESCRIPTION
**Pull request by manasa123104**

### CODE REVIEW

**Comment by gokulprathin8** [comment]
please merge fSCA training and testing into a single chapter

**Comment by saivivek116** [comment]
can you pls check the markdown of the headings
there is not separation between heading and its contents 
use bold text for headings and use : or new line for seperation
for proper markdown usage
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax
refer this 
and review your changes in the preview

**Comment by saivivek116** [comment]
No Code related to fSCA training, fSCA testing and data_snotel_stations process nodes in the books.
can you pls add Code and its explanation for the better understanding.

File: book/chapters/fsCA.ipynb [hunk 1]
```diff
@@ -2,32 +2,203 @@
...
-   "metadata": {
-    "collapsed": false
-   },
-   "id": "83c689bcf816fa42"
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "\n",
+    "models = ['SVM', 'RF', 'CNN', 'RNN']\n",
+    "accuracies = [82, 85, 92, 90]\n",
```
Comment (ZihengSun) [inline book/chapters/fsCA.ipynb]: What do these numbers come from? It feels disrupted coming out from nowhere.

**Review by ZihengSun — CHANGES_REQUESTED** [review]
The markdown is very messy. Please correct. Also, get the chapter logic together. It doesn't make any sense right now.

**Comment by gokulprathin8** [comment]
snotel review:
[snotel.pdf](https://github.com/geo-smart/swe-workflow-book/files/14860012/snotel.pdf)

**Comment by gokulprathin8** [comment]
fSCA review:
[fsCA.pdf](https://github.com/geo-smart/swe-workflow-book/files/14860228/fsCA.pdf)

**Comment by ZihengSun** [comment]
probably shouldn't remove the three data files.

**Pull request closed without merging**